core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -170,16 +186,23 @@ pub async fn check_tx(tx_hash: &str) -> Option<bool> {
 //     }
 // }
 
-fn get_provider() -> Provider {
-    (*PROVIDER).clone()
+fn get_provider(network: Network) -> Provider {
+    let provider: Provider = match env::var("ZKSYNC_RPC_ADDRESS").ok() {
```

Comment:
I think we should no longer have a single variable for zkSync RPC address but rather separate variables for every network.

Reply:
good idea, but i propose to create an issue for this and move on with finishing `fund`

Reply:
OK